ci/jenkins/test.sh

```diff
@@ -989,6 +1012,10 @@ if [[ $TESTCASE =~ "multicast" ]]; then
 fi
 echo $MANIFEST_ARGS
 
+if [[ $TESTBED_TYPE == "kind-flexible-ipam" ]]; then
+    MANIFEST_ARGS="$MANIFEST_ARGS --flexible-ipam --multicast --verbose-log"
```

Comment:
May I know why the flag `multicast` is needed?

Reply:
to run ACNPAntreaIPAMMulticast tests :
### --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast (106.16s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPIGMPQueryAllow (40.03s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPIGMPQueryDrop (34.88s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPPolicyEgressAllow (13.83s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPPolicyEgressDrop (17.42s)


we need to run these tests right?

Reply:
I see, thanks for the explaination.